### PR TITLE
configureStore merge

### PR DESCRIPTION
**Pull request by goldoraf**

### CODE REVIEW

File: src/drive/mobile/lib/reporter.js [hunk 1]
```diff
@@ -1,21 +1,19 @@
 /* global __SENTRY_TOKEN__, __DEVELOPMENT__ */
 import Raven from 'raven-js'
 
-let isEnabled = false
```
Comment (enguerran) [inline src/drive/mobile/lib/reporter.js]: Remove this causes an error.

**Comment by enguerran** [comment]
Aside from a full bench of tests, I cannot see if this PR causes regressions.

**Comment by y-lohse** [comment]
Same here, too many changes to really understand what's going on with only the diff. I would even have missed the `isEnabled` thing that @enguerran pointed out.
I did a quick check out and of the PR and toured the app, didn't see anything broken.

**Comment by enguerran** [comment]
J'ai cette erreur : 


Pour reproduire : 

```
git checkout 6def9001
rm -rf node_modules
yarn
yarn watch:drive:standalone
open localhost:8084
```

**Pull request closed without merging**